### ISSUE TITLE
Wpf: OnMainFormChanged

### ISSUE THREAD

**Issue by TomQv** [picoe/Eto#816]
On Wpf closing a splash-screen, which is the first form of the app, causes the app to close, because changing MainForm-property to MyMainForm doesn't have an effect.

Fix:
Eto.Wpf.Forms.ApplicationHandler.OnMainFormChanged
```
public void OnMainFormChanged()
{
   if (Widget.MainForm!=null)
         System.Windows.Application.Current.MainWindow =Widget.MainForm.ToNative ();
}

```

**Comment by msasso69** [comment]
Hi, how do you manage to position your splash-screen at the center of the screen?

**Comment by cwensley** [comment]
Fixed in #978
